scipy/stats/tests/common_tests.py

```diff
@@ -292,6 +292,19 @@ def check_pickling(distfn, args):
     distfn.random_state = rndm
 
 
+def check_freezing(distfn, args):
+    # regression test for gh-11089: freezing a distribution fails
+    # if loc and/or scale are specified
+    if isinstance(distfn, stats.rv_continuous):
+        locscale = {'loc': 1, 'scale': 2}
+    else:
+        locscale = {'loc': 1}
+
+    rv = distfn(*args, **locscale)
+    assert rv.a == distfn(*args).a
+    assert rv.b == distfn(*args).b
```

Comment:
@rlucas7 @ev-br Currently, this test passes because the `a` and `b` attributes of a frozen distribution are not affected by the location and scale. This behavior was reported as a bug in gh-13225, and I tend to agree - if the distribution is frozen, the shape, location, and scale affect the support, and `a` and `b` should reflect that. I tried to fix this in gh-13228, but this test started to fail.  Was this intentional - do you think that the frozen distribution's `a` and `b` should report the support of the unscaled, unshifted distribution?